api/app/settings/common.py

```diff
@@ -460,3 +462,20 @@
 # Set this flag to prevent traits being stored for all Organisations within the application
 # Useful for data sensitive installations that dont want persistent traits.
 DEFAULT_ORG_STORE_TRAITS_VALUE = env.bool("DEFAULT_ORG_STORE_TRAITS_VALUE", True)
+
+
+### Front end environment variables
```

Comment:
This will need the PAYMENTS_ENABLED env far https://github.com/Flagsmith/flagsmith/commit/61887b7b6911b2a752fa5907559a09e2790af630
https://github.com/Flagsmith/flagsmith/issues/134

Reply:
No longer needed: PAYMENTS_ENABLED is now a feature flag